src/aerie_cli/utils/configurations.py

```diff
@@ -0,0 +1,29 @@
+from aerie_cli.aerie_host import AerieHostConfiguration
+from aerie_cli.persistent import PersistentConfigurationManager
+
+import json
+
+def find_configuration(configuration: str) -> AerieHostConfiguration:
+    """Find a configuration by name or path.
+    
+    configuration (str): The name or path of a configuration. Paths should be to a configuration json file.
+
+    Returns:
+        AerieHostConfiguration
+    """
+    # search for configuration by name
+    for persistent_configuration in PersistentConfigurationManager.get_configurations():
+        if persistent_configuration.name != configuration:
+            continue
+        return persistent_configuration
+    # search for configuration by path
+    try:
+        with open(configuration, 'r') as fid:
+            try:
+                found_configuration = json.load(fid)
+            except ValueError as e:
+                return None
+    except FileNotFoundError as e:
+        return None
```

Comment:
That makes sense, I believe I've addressed this now with 0322bde. I'm catching the ValueError in main because it would be a very cryptic error message otherwise.

Reply:
Ah, I meant to raise the exceptions in `find_configuration`

Reply:
Oooh ok ok I understand why what I said is confusing, sorry! My suggestion is that we catch the `ValueError` and `FileNotFoundError` in `find_configuration`, but then re-raise them (like you did in main) with the additional explainer message.

Reply:
Bingo, that looks great now. Thanks!

Reply:
I changed the exceptions to be raised in `find_configurations` in 0322bde, but started handling the errors outside of it. I see that I ended up raising them again in main again when trying to give a cleaner error message, so now I'm raising the errors with the relevant messages in `find_configurations` in 26a6cd9.